src/others/components/LanguageSelector.tsx

```diff
@@ -51,11 +54,24 @@ export const LanguageSelector: React.FunctionComponent<LanguageSelectorProps> =
     setExpanded(false);
   };
 
+  const handleClick = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setExpanded((prevState) => !prevState);
+  };
```

Comment:
Thanks! You are right 👍 
I've used prevState for this reason just in case https://dev.to/nicolelc16/react-previous-state-4akk.
But I agree with you (since we are just changing from true <-> false, strictly no need here)

Reply:
My snippet is missing the import of `useCallback` 😅

Reply:
Ah I now get the `prevState`. I misread it 😄 This also should remove it from the dependencies in the useCallback Hook